Oh, it might actually be this one!
```
./addons/mrp/migrations/10.0.2.0/post-migration.py:21:1: E302 expected 2 blank lines, found 1
./addons/mrp/migrations/10.0.2.0/post-migration.py:24:9: E128 continuation line under-indented for visual indent
./addons/mrp/migrations/10.0.2.0/post-migration.py:48:13: W291 trailing whitespace
./addons/mrp/migrations/10.0.2.0/post-migration.py:52:14: W291 trailing whitespace
./addons/mrp/migrations/10.0.2.0/post-migration.py:57:5: E124 closing bracket does not match visual indentation
./addons/mrp/migrations/10.0.2.0/post-migration.py:60:9: E128 continuation line under-indented for visual indent
./addons/mrp/migrations/10.0.2.0/post-migration.py:83:14: W291 trailing whitespace
./addons/mrp/migrations/10.0.2.0/post-migration.py:88:5: E124 closing bracket does not match visual indentation
./addons/mrp/migrations/10.0.2.0/post-migration.py:89:1: W293 blank line contains whitespace
The command "flake8 --max-line-length=120 $(find . \( -name 'pre-*.py' -or -name 'post-*.py' \) -exec grep -q openupgrade {} \; -print)" exited with 1.
```

--- a/addons/mrp/migrations/10.0.2.0/post-migration.py
+++ b/addons/mrp/migrations/10.0.2.0/post-migration.py
@@ -45,11 +45,11 @@ def populate_stock_move_lots(cr):
           sm.restrict_lot_id,
           TRUE AS done_move,
           sm2.restrict_lot_id AS lot_produced_id
-        FROM 
+        FROM
           stock_move AS sm
         INNER JOIN
           stock_move AS sm2 ON sm.%s = sm2.id
-        WHERE 
+        WHERE
           sm.restrict_lot_id IS NOT NULL
           AND sm.raw_material_production_id IS NOT NULL
           AND sm.state = 'done'
@@ -80,13 +80,13 @@ def populate_stock_move_lots(cr):
           restrict_lot_id,
           TRUE as done_move
         FROM stock_move
-        WHERE 
+        WHERE
           restrict_lot_id IS NOT NULL
           AND production_id IS NOT NULL
           AND state = 'done'
         """
     )
-    
+
 
 def archive_mrp_bom_date_stop(cr):
     cr.execute(

